src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -1216,6 +1216,22 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
         tmp = const_tuple;
     }
 
+    void visit_IntegerBitLen(const ASR::IntegerBitLen_t& x) {
+        this->visit_expr_wrapper(x.m_a, false);
+        llvm::Value *int_val = tmp;
+        std::string runtime_func_name = "_lpython_bit_length";
+        llvm::Function *fn = module->getFunction(runtime_func_name);
+        if (!fn) {
+            llvm::FunctionType *function_type = llvm::FunctionType::get(
+                    llvm::Type::getInt32Ty(context), {
```

Comment:
This won't work for this test:
```python
i: i8
i = 423
assert i.bit_length() == 9
```
This test should be added in `test_bit_length.py`.

Reply:
What's is the best approach for that, previously I made for every `int` kind C functions.